root/static/scripts/edit/ExampleRelationships.js

```diff
@@ -10,13 +10,13 @@ MB.ExampleRelationshipsEditor = (function (ERE) {
 
 
 // Private variables
-var type0, type1, linkTypeName, linkTypeID, jsRoot;
+var type0; var type1; var linkTypeName; var linkTypeID; var jsRoot;
```

Comment:
These should be in a new line each :)

There's a bunch more below, so you can probably just find them by searching for "; var"